tools/cli/commands-packages.js

```diff
@@ -1649,6 +1649,22 @@ main.registerCommand({
   } else {
     upgradePackageNames = options.args;
   }
+  var upgradePackagesWithoutCordova = _.reject(upgradePackageNames, function (name) {
+    return name.split(':')[0] === 'cordova'; 
+  });
+  if (upgradePackagesWithoutCordova !== upgradePackageNames) {
+    // There are some cordova packages in the list to update.
+    // We should tell users how to update cordova packages.
+    Console.warn('Looks like you\'re trying to update cordova packages.');
```

Comment:
Additionally, maybe this should be rephrased to:

    It appears that you're trying to upgrade a Cordova package.  To perform this action, you should run: